nimblepayment.php

```diff
@@ -180,7 +180,7 @@ public function renderForm()
 				),
 				array(
 						'id_option' => 'sandbox',
-						'name' => 'Sandbox'
+						'name' => 'Pruebas'
```

Comment:
I guess we should add the translation system here, in order to display the English version if needed:
```php
'name' => $this->l('Pruesbas')
```